content/blog/hpe-athonet-llm-platform-first-pillar-from-personal-assistant-to-collaborative-corporate-tool.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "HPE Athonet LLM Platform, first pillar: from personal assistant to
```

Comment:
Hello, Antonio,
Dale here... your friendly blog editor ;-) 
I was wondering, do you really need "first pillar" in the title. I know a lot of our blog posts have long titles, and I'm trying to encourage everyone to move to shorter ones. It makes it so much easier to include the titles in the social media posts if you do, since there's a character count limit on those.

In addition, I'm not sure it really adds anything in the title. Wouldn't it just work as:
"HPE Athonet LLM platform: From personal assistant to collaborative corporate tool" [after a colon in a title or a header, you can make the next word an initial capital letter]